src/ast/mod.rs

```diff
@@ -252,6 +252,11 @@ pub enum Expr {
         field: DateTimeField,
         expr: Box<Expr>,
     },
+    /// POSITION(<expr> in <expr>)
+    Position {
+        expr: Box<Expr>,
+        from: Box<Expr>,
```

Comment:
I wonder if calling this field `in` (rather than `from`) would be better as it is closer to the original SQL text (which uses `in`)?

Reply:
@alamb I called it from because I found it very similar to the `substring`  function. I think `in` is a great option